config.yaml

```diff
@@ -11,6 +11,7 @@ enterpriseAssistant:
   eaAddress: '' # Address of the EA server (example: https://<your EA Address>:8000)
   eaUsername: '' # Username for the EA server given in EA Settings
   eaPassword: '' # Password for the EA server given in EA Settings
+wifiSynchEnabled: true
```

Comment:
this shoudl be wifiSyncEnabled not wifiSynchEnabled

Reply:
:-) updated.